docs/v1/src/pages/docs/react/SuspensiveProvider.en.mdx

```diff
@@ -2,7 +2,7 @@
 
 This component controls the settings of the components provided by Suspensive (Suspense, Delay, etc.) at once.
 
-```tsx /SuspensiveProvider/ /Suspensive/
+```jsx /SuspensiveProvider/ /Suspensive/
```

Comment:
Why did we switch to JSX?

Reply:
Unfortunately, prettier don't make format it in tsx